frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template-create-yaml.tsx

```diff
@@ -22,13 +23,38 @@ const CreateVMTemplateYAMLConnected = connectToPlural(
       }
       return <ErrorPage404 />;
     }
+    const [selectedTemplate, setTemplate] = React.useState(null);
+    const [templateLabel, setTemplateLabel] = React.useState(null);
 
     const template = VMTemplateYAMLTemplates.getIn(['vm-template']);
     const obj = safeLoad(template);
     obj.kind = kindObj.kind;
     obj.metadata = obj.metadata || {};
     obj.metadata.namespace = match.params.ns || 'default';
 
+    if (templateLabel) {
+      obj.objects[0].metadata.labels['vm.kubevirt.io/template'] = templateLabel;
```

Comment:
please use constants for these -  please see `constants/vm` or/and add the ones missing